news/_posts/2021-06-24-Bottenhorn-defense.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: news
+title: Katie Bottenhorn defends dissertation
+author: Taylor Salo
+category: news
+tags: []
+image: /assets/images/team/katherine-bottenhorn.jpg
+excerpt: Graduate student Katie Bottenhorn has successfully defended her dissertation
+---
+
+[Katie Bottenhorn](/team/bottenhorn-katherine) successfully defended her dissertation,
+"Development and Deployment of Analytic Tools for Interrogating Large-scale Brain Networks", on June 24, 2021,
+and has now received her Ph.D. in Cognitive Neuroscience.
+
+Katie will soon start a post-doc at the University of Southern California in
+Dr. Megan Herting's [Herting Neuroimaging Lab](https://hertinglab.usc.edu).
+
+Congratulations Katie!
```

Comment:
@62442katieb is there anything you'd like to see changed here to make it more customized? Otherwise it looks really similar to Jessica and Ranjita's announcements.

Reply:
Oops! I just saw this comment.; will make a PR shortly.